recipe/meta.yaml

```diff
@@ -55,6 +55,7 @@ outputs:
         - {{ compiler('c') }}                # [not win]
         - {{ compiler('cxx') }}              # [not win]
         - {{ compiler('fortran') }}          # [not win]
+        - gfortran_impl_linux-64 {{ fortran_compiler_version }}  # [ppc64le or aarch64]
```

Comment:
Where is this used?

Reply:
In compiling `r-base` for the `build-platform`.

Reply:
```suggestion
        - gfortran_impl_{{ build_platform }} {{ fortran_compiler_version }}  # [build_platform != target_platform]
```